fivex/api/format.py

```diff
@@ -215,6 +215,49 @@ def position_to_variant_id(
     return f"{chromosome}:{position}_{ref_allele}/{alt_allele}"
 
 
+@dc.dataclass
+class CIContainer:
+    """
+    Represents the data for credible intervals
+    """
+    # Study and tissue are not present in study- and tissue-specific files -- these two fields are only present in merged files
+    study: str
+    tissue: str
+
+    # The rest of these fields are present in all credible interval files
+    gene_id: str  # this column is labeled "phenotype_id" in the original file
+    var_id: str  # in chrom_pos_ref_alt format -- not used
```

Comment:
Future note: I've been trying to reduce the number of different places (and ways) that we calculate variant IDs, as part of ongoing efforts to harmonize common information to a common representation.

There's a new helper function in format.py that uses EPACTS style `chrom:pos_ref/alt` (with comma separators on position, for human readability); see `VariantContainer` for how we add that to the dataclass.